src/main/java/tr/com/srdc/cda2fhir/transform/entry/CDAIIResourceMaps.java

```diff
@@ -0,0 +1,108 @@
+package tr.com.srdc.cda2fhir.transform.entry;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.openhealthtools.mdht.uml.hl7.datatypes.II;
+
+import tr.com.srdc.cda2fhir.transform.util.ICDAIIResourceMaps;
+import tr.com.srdc.cda2fhir.transform.util.ICDAIIResourceMapsSource;
+import tr.com.srdc.cda2fhir.transform.util.impl.CDAIIMap;
+
+public class CDAIIResourceMaps<T> implements ICDAIIResourceMaps<T>, ICDAIIResourceMapsSource<T> {
+	Map<Class<? extends T>, CDAIIMap<T>> iiMaps = new HashMap<Class<? extends T>, CDAIIMap<T>>();
```

Comment:
Are these resourceMaps used somewhere or are they for future?

Reply:
The plan is to use them for all resources that need to be deduped with IIs